Jenkinsfile

```diff
@@ -203,7 +203,7 @@ pipeline {
                 stage("Dune tests") {
                     agent {
                         docker {
-                            image 'stanorg/stanc3:debianfi'
+                            image 'stanorg/stanc3:debianfi-codecov'
```

Comment:
This will be changed back once @serban-nicusor-toptal updates the base docker image (Thanks Nic!)